README.rst

```diff
@@ -8,8 +8,8 @@ bg-mpl-stylesheets
 .. |Black| image:: https://img.shields.io/badge/code_style-black-black
         :target: https://github.com/psf/black
 
-.. |Codecov| image:: https://codecov.io/gh/bg-mpl-stylesheets/bg-mpl-stylesheets/branch/main/graph/badge.svg
-        :target: https://codecov.io/gh/bg-mpl-stylesheets/bg-mpl-stylesheets
+.. |Codecov| image:: https://codecov.io/gh/Billingegroup/bg-mpl-stylesheets/branch/main/graph/badge.svg
```

Comment:
Many URLs didn't work as the organization was moved from `bg-mpl-stylesheets` to `Billingegroup`.

Reply:
thanks for catching and fixing that.